### ISSUE TITLE
Puppet Unknown variable: `reboot`

### ISSUE THREAD

**Issue by jcpunk** [puppetlabs/puppetlabs-patching_as_code#55]
## Describe the Bug
There appears to be no value set to `$reboot` under one path.

```
[puppetserver] Puppet Unknown variable: 'reboot'. (file: /etc/puppetlabs/code/environments/production/r10k/patching_as_code/functions/process_patch_groups.pp, line: 75, column: 33)
```

## Expected Behavior
Variables should be defined before use

## Steps to Reproduce
1. Define a custom group
2. run puppet on "not patch day"
3. review server logs

## Environment
 - Version 1.1.2
 - Platform RHEL8

## Additional Context
I think an `else` condition is needed at 
https://github.com/puppetlabs/puppetlabs-patching_as_code/blob/v1.1.2/functions/process_patch_groups.pp#L41

but I'm unsure what the value of `$reboot` should be under that branch.

**Comment by kreeuwijk** [comment]
Yes, I'm aware. It's a benign error so you can ignore it for now.
I'll put in a handler for this particular case.

**Comment by kreeuwijk** [comment]
Fixed in v1.1.3 which is now available on the Forge
